Source/engine/render/text_render.cpp

```diff
@@ -275,6 +275,11 @@ void WordWrapGameString(char *text, size_t width, GameFontTables size, int spaci
 	}
 }
 
+int DrawString(const CelOutputBuffer &out, const char *text, const Point &position, uint16_t flags, int spacing, int lineHeight, bool drawTextCursor)

```

Comment:
Normally this kind of function would be defined in the header, so that the compiler can elide the call even without LTO

Reply:
Done :)